google_books/src/data/apiDataFunctions.js

```diff
@@ -0,0 +1,36 @@
+const basicSearch = async (
+  searchTerm = "",
+  title = "",
+  author = "",
+  publisher = "",
+  resPerPg = 40
+) => {
+  const titleVal = title !== "" ? `+intitle:${title}` : ``;
+  const authorVal = author !== "" ? `+inauthor:${author}` : ``;
+  const publisherVal = publisher !== "" ? `+inpublisher:${publisher}` : ``;
+
```

Comment:
Sanitize the terms coming in, searchTerms that contain `&` or `?` might break your query